contracts/v1/OpenOracleV1.sol

```diff
@@ -0,0 +1,44 @@
+// SPDX-License-Identifier: GPL-3.0
+
+pragma solidity ^0.6.10;
+pragma experimental ABIEncoderV2;
+
+import "../Uniswap/UniswapAnchoredView.sol";
+
+interface IPriceOracle {
+	function assetPrices(address) external returns (uint256);
+	function getPrices(address) external returns (uint256);
+}
+
+/**
+ * @title Open Oracle Adapater for v1 Interface
+ * @author Compound Labs, Inc.
+ */
+contract OpenOracleV1 is IPriceOracle {
+	UniswapAnchoredView public immutable openOracleView;
+
+	constructor(UniswapAnchoredView openOracleView_) public {
+		openOracleView = openOracleView_;
+	}
+
+	function getPrices(address asset) public override returns (uint256) {
+		UniswapConfig.TokenConfig memory ethConfig = openOracleView.getTokenConfigBySymbol("ETH");
```

Comment:
holy crow, good thing those get configs were public

Reply:
Yeah, otherwise this would have been way more complex to build